test_e2e.sh

```diff
@@ -1066,12 +1066,12 @@ function e2e::askpass_url() {
         --root="$ROOT" \
         --link="link" \
         >> "$1" 2>&1 || true
-    docker_kill "$CTR"
 
     # check for failure
     assert_file_absent "$ROOT"/link/file
 
     # run with askpass_url service with correct password
+    docker_kill "$CTR"
```

Comment:
Why the manual kill here? To force a kill before CTR gets rewritten?

Reply:
correct - CTR is a helped and I need to kill and restart it here